test/integration/lib/philipsHue_server.js

```diff
@@ -107,14 +108,20 @@ var philipsHue_server = (function() {
         res.status(200).json(response);     
       });
 
-      _server.listen(port, function () {
-        console.log('Hue simulator listening on port ' + port);
-      });
-    }
+    instance = _server.listen(port, function () {
+      console.log('Hue simulator listening on port ' + port);
+    });
+  }
+
+  function stop() {
```

Comment:
Nit: Same comment as [this one](https://github.com/fxbox/foxbox/pull/235/files#r57879230)